backend/src/backend/routes/netjson.clj

```diff
@@ -67,11 +67,40 @@
                                     (params "dateto")))
     :else (error-handler-rep 400 "Invalid query")))
 
+(defn parse-int [s]
+  (Integer. (re-find  #"\d+" s)))
```

Comment:
Is there a reason that we are not using read-string for this?

Reply:
Not really. I'll change that.